libs/wingc/src/type_check.rs

```diff
@@ -166,6 +166,8 @@ pub enum Type {
 pub const CLASS_INIT_NAME: &'static str = "init";
 pub const CLASS_INFLIGHT_INIT_NAME: &'static str = "$inflight_init";
 
+pub const HANDLE_METHOD_NAME: &'static str = "handle";
```

Comment:
Should we make this `$handle` so users can't create their own "handler" classes? I'm worried this name will start to collide with stuff, "handler" is pretty common?

Reply:
I feel like it's pretty common to use well-defined / common names for important language features. For example, consider how "self" is a special argument when it's the name of the first parameter of a method in Rust. We're basically treating "handle" with the same level of importance here.

Reply:
I think that if we currently name it something that the user can never create on their own (`$handle`) then we might be able to discuss/change this later. By keeping this `handle` we're kind of creating a language feature (like `self`).
I'd like some more opinions here: @eladb?

Reply:
I am not 100% sure whether `handle` is the right now for the method name. In Java it could be any name and the generated class will take the name based on the interface it is implementing.

But, I'd like users to be able to simply pass an object that implements the interface as an alternative to passing a closure.

Bottom line, My preference is to leaving it `handle` for now.

Reply:
Sounds good 👍